vendor/index.yaml

```diff
@@ -1649,3 +1649,14 @@ vendors:
     name: ACRIOS Systems s.r.o.
     website: https://acrios.com/
     logo: acrioslogo.png
+
+  - # Unique identifier of the vendor (lowercase, alphanumeric with dashes, max 36 characters)
```

Comment:
Hello @TD-er please take out commented lines, to have the same format as other device makers. Thank you

Reply:
Done.
I also removed it from another entry a few lines above.